hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/states/ContainerStateMap.java

```diff
@@ -128,6 +124,21 @@ public void addContainer(final ContainerInfo info)
     }
   }
 
+  /**
+   * The idea here is to rebuild the containerStateMap once the SCM StateMachine
+   * reloads from the the DBCheckpoint downloaded from leader SCM.
+   *
+   * Replica Map won't be rebuild but will remain as it is.
+   */
+  public void build(final ContainerInfo info) throws SCMException {
```

Comment:
Will it be used at other place than `addContainer` ?

Reply:
NO, it won't be.